[inductor] More tweaks to fusion logs

int3 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #115084

I think it's more useful to print out actual fusions rather than
possible fusions.

I also updated `speedup_by_fusion`'s logs to include the node names in
the log output.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

int3 committed:
[inductor] More tweaks to fusion logs

I think it's more useful to print out actual fusions rather than
possible fusions.

I also updated `speedup_by_fusion`'s logs to include the node names in
the log output.

[ghstack-poisoned]

ipiszy commented on torch/_inductor/scheduler.py:
```diff
@@ -1772,11 +1769,6 @@ def check_all_pairs(nodes):
                 check_all_pairs(node_grouping)
 
         possible_fusions.sort(key=self.score_fusion_key, reverse=True)
-        if fusion_log.isEnabledFor(logging.DEBUG):
```
I feel it's still useful to keep them for debugging purpose.

int3 replied:
I think it's redundant since the nodes that don't get fused will get logged anyway in `speedup_by_fusion`. Every possible fusion will already have at least one log line printed out, either to say we are fusing them, or why we are not fusing them.

ipiszy replied:
Is it possible that some fusions are not logged because priorities are low? i.e. High-pri fusions are already done so low-pri fusions are ignored.

int3 replied:
I don't think so... `can_fuse` and `will_fusion_create_cycle` also print out a log if they determine a fusion is not possible due to changes in the graph from higher-pri fusions.

ipiszy reviewed (commented):
Thanks @int3 !

ipiszy commented on torch/_inductor/scheduler.py:
```diff
@@ -1772,11 +1769,6 @@ def check_all_pairs(nodes):
 
         possible_fusions.sort(key=self.score_fusion_key, reverse=True)
-        if fusion_log.isEnabledFor(logging.DEBUG):
-            fusion_log.debug("\nfound %d possible fusions:", len(possible_fusions))
```
At least we want to preserve this?

int3 committed:
Update on "[inductor] More tweaks to fusion logs"


I think it's more useful to print out actual fusions rather than
possible fusions.

I also updated `speedup_by_fusion`'s logs to include the node names in
the log output.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

[ghstack-poisoned]

int3 commented:
@pytorchbot merge
